e2e/autoscript-suites/src/suites/authz-tree-basic-redirect.test.ts

```diff
@@ -11,9 +11,9 @@
 import { test, expect } from '@playwright/test';
 import { setupAndGo } from '../utilities/setup-and-go';
 
-test.describe('Test Transaction Authorization flow', () => {
-  test(`Trigger Txn Auth appropriately`, async ({ page, browserName }) => {
-    const { messageArray } = await setupAndGo(page, browserName, 'authz-txn-basic/');
+test.describe('Test Tree Based Authorization flow using Redirect response', () => {
+  test(`Trigger Tree Based Auth appropriately`, async ({ page, browserName }) => {
+    const { messageArray } = await setupAndGo(page, browserName, 'authz-tree-basic-redirect/');
```

Comment:
This is an example of the test fixes for transactional / tree-based authorization that I feel are necessary; here we had the Tree Based test referencing the setup for the Transactional test app. This means that certain responses would not have been tested.